### PR TITLE
DarkiStar firmware patch

### PR DESCRIPTION
**Pull request by DarkiStar**
| Q             | A
| ------------- | ---
| Bug fix?      | yes
| New feature?  | no
| Doc update?   | yes
| BC breaks?    | no 
| Deprecations? | no
| Fixed tickets | #...

### CODE REVIEW

**Comment by DarkiStar** [comment]
This patch is intended to solve the ominous problem of displaying invalid firmware versions. Boards which show a firmware version 0x12, 0x80, 0x00 etc. are therefore not defective. There seems to be another problem with DumpInfo.ino. Some readers will only read the tags if they are within range of the reader when starting the SerialMonitor.

**Comment by Rotzbua** [comment]
Your solution implies, that the pcd is not ready after the init process. Shouldnt be the delay in the init process?

File: README.rst [hunk 1]
```diff
@@ -242,11 +242,10 @@ Troubleshooting
   #. According to reports #101, #126 and #131, there may be a problem with the soldering on the MFRC522 breakout. You could fix this on your own.
 
 
-* **Firmware Version: 0x12 = (unknown)**
+* **Firmware Version: 0x12 = (unknown) and 0x80 = (unknown) and other random values**
 
-  #. The reason of this behaviour is unknown.
-  #. If this sometimes appear a bad connection or power source is the reason.
```
Comment (Rotzbua) [inline README.rst]: this was observed in the past

File: README.rst [hunk 2]
```diff
@@ -242,11 +242,10 @@ Troubleshooting
...
-  #. If the firmware version is reported permanent, it is very likely that the hardware is a fake or has a defect. Contact your supplier.
+  #. The reason of this behaviour is due to a direct call of PCD_DumpVersionToSerial() without delay() after PCD_Init()
+  #. a simple delay(5) leads to correct firmware readings (see updated example DumpInfo)
```
Comment (Rotzbua) [inline README.rst]: example `delay(4);    ` ?

File: README.rst [hunk 3]
```diff
@@ -242,11 +242,10 @@ Troubleshooting
   #. According to reports #101, #126 and #131, there may be a problem with the soldering on the MFRC522 breakout. You could fix this on your own.
 
 
-* **Firmware Version: 0x12 = (unknown)**
+* **Firmware Version: 0x12 = (unknown) and 0x80 = (unknown) and other random values**
 
-  #. The reason of this behaviour is unknown.
-  #. If this sometimes appear a bad connection or power source is the reason.
-  #. If the firmware version is reported permanent, it is very likely that the hardware is a fake or has a defect. Contact your supplier.
```
Comment (Rotzbua) [inline README.rst]: still valid

File: README.rst (hunk 2 above)
Comment (DarkiStar) [reply README.rst]: In my tests any value above 3 worked. How can I change my PR?

File: README.rst (hunk 3 above)
Comment (DarkiStar) [reply README.rst]: Ok.

File: README.rst (hunk 2 above)
Comment (Rotzbua) [reply README.rst]: change the files in your branch `DarkiStar:DarkiStar-Firmware-Patch`

**Comment by Rotzbua** [comment]
Thx. I took your pr to d36f9370ea3b075c4b9730388ce620bc7f28f929.

**Comment by Kinko530** [comment]
Try connecting the SDA pin directly to an SDA pin on the uno/mega.

**Pull request closed without merging**